main.py

```diff
@@ -350,7 +313,4 @@ def resource_gone(e: BaseException) -> Tuple[Response, int]:
     )
     app.logger.info(log_str)  # type: ignore
 
-    # Pre-load the correction engine into memory
```

Comment:
This was there to make the first request response time a bit lower, while taking a tiny bit more time to initialize the server.